score/etl/base.py

```diff
@@ -0,0 +1,62 @@
+from pathlib import Path
+import pathlib
+
+from config import settings
+from utils import unzip_file_from_url, remove_all_from_dir
+
+
+class ExtractTransformLoad(object):
+    """
+    A class used to instantiate an ETL object to retrieve and process data from
+    datasets.
+
+    Attributes:
+        DATA_PATH (pathlib.Path): Local path where all data will be stored
+        TMP_PATH (pathlib.Path): Local path where temporary data will be stored
+        GEOID_FIELD_NAME (str): The common column name for a Census Block Group identifier
+        GEOID_TRACT_FIELD_NAME (str): The common column name for a Census Tract identifier
+
+    Methods:
+        get_yaml_config() -> None: Reads the YAML configuration file for the dataset and stores
```

Comment:
Nit: is there a reason we're writing these here, rather than writing docstrings on the method itself? 

E.g., as: 

``` 
def get_yaml_config() -> None: 
    """Reads the YAML configuration file for the dataset and stores"""
``` 

I've usually done it that way!

Reply:
I saw an example of both. A positive of doing it that way is that the methods appear on the class hint, but let me try the other way.